pkg/apis/auditregistration/v1alpha1/defaults.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	auditregistrationv1alpha1 "k8s.io/api/auditregistration/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	// DefaultQPS is the default QPS value
+	DefaultQPS = func(i int64) *int64 { return &i }(int64(10))
```

Comment:
Something got scrambled, no doubt in a comment I gave :) 

I suggest making these consts, declaring `func intPtr(i int64) *int64 { return &i }` somewhere, and calling `intPtr(DefaultQPS)` in the `DefaultThrottle` function.

Making these constant will prevent people from taking their address, which is the error I think it's important to avoid.